src/operator-docs/transformation/windowCount.ts

```diff
@@ -1,6 +1,77 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowCount: OperatorDoc = {
-  'name': 'windowCount',
-  'operatorType': 'transformation'
+  name: 'windowCount',
+  operatorType: 'transformation',
+  signature:
+    'public windowCount(windowSize: number, startWindowEvery: number): Observable<Observable<T>>',
```

Comment:
Please remove `<T>` we have decided not to use generics.

Reply:
Yep...